room_assistance/indico_room_assistance/client/style.scss

```diff
@@ -0,0 +1,32 @@
+// This file is part of the CERN Indico plugins.
+// Copyright (C) 2014 - 2019 CERN
+//
+// The CERN Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License; see
+// the LICENSE file for more details.
+
+.assistance-occurrences-widget {
+  display: flex;
```

Comment:
we only use 2-space indentation for JS ;)

Reply:
Ah, now I see why it seemed off to me as well. Sass-lint reported that I should be using 2-space  indentation which is the default I guess. We are not using our `.sass-lint.yml` file when running `sass-lint` over plugins.